cypher-manual/overview.md

```diff
@@ -82,6 +82,11 @@ of your data, but be careful with this if you don't have a backup!
 Maybe [clauses](clauses/clauses.md) are the reason why you are here. We have
 created a list of all Cypher clauses that Memgraph supports.
 
+## Memgraph extension clauses
+
+Memgraph extension clauses (for example, foreach), are additional clauses added
```

Comment:
```suggestion
Memgraph extension clauses (for example, `FOREACH`), are additional clauses added
```

Also I am not sure if we should call it _Memgraph extension caluses_ as it was "introduced" by Neo4j. Maybe just call it _Extensions_ or _Clauses on top of openCypher` or something similar.